### ISSUE TITLE
Support more then one Subscriber

### ISSUE THREAD

**Issue by philolo1** [callstack/react-native-opentok#51]
I dont think you support yet more then one subscriber, i need this feature and will do a pull request for this. I think for this purpose the subscriberview should get a streamId property.

**Comment by loctrandn** [comment]
+1
I need support multi subscriber

**Comment by dratwas** [comment]
@philolo1 What you mean about multiple subscribers?
This library supports multiple session so you can connect with two sessions and then render two subscribers view with different `sessionId` prop.

Are you using few streams in one session?

**Comment by RoyalDeveloper2015** [comment]
can I test Publisher & Subscriber in one Component at the same time with . one Session ID ?

**Comment by dratwas** [comment]
Nope, because we're connecting with a session using `token` which is related to subscriber or publisher. I'm thinking if we can change our implementation, but Is it needed for you or it is just your test case?

**Comment by RoyalDeveloper2015** [comment]
I'd like to build app similar to skype , there you can see you your video and others video .
how can we implement like that using this SDK ?

**Comment by dratwas** [comment]
I'm not sure if it is possible. I have to check. At this moment you can't achieve that with this library.

**Comment by philolo1** [comment]
its possible, you just need to use the streams instead of the sessions, i
forked the library and made it worked, but my solution is not to good to
merge it or do a pull request.

2017-10-31 19:09 GMT+09:00 Drapich Piotr <notifications@github.com>:

**Comment by dratwas** [comment]
Ok, so I think we will implement this feature in next releases :)

**Comment by alexandermtang** [comment]
@philolo1 would you be able to share your fork that uses streams instead of sessions?

**Comment by siliconavengers** [comment]
@philolo1 can you share your fork? We're facing the same issue, when we want to subscribe multiple streams
